working/0723-static-extensions/feature-specification.md

```diff
@@ -315,165 +453,482 @@ extension.
 
 #### Declarations of constructors in extensions
 
-This proposal relies on the [generic constructor proposal][]. In
-particular, this proposal uses concepts and definitions from the generic
-constructor proposal, and it is assumed that generic constructors are
-supported by the underlying Dart language.
+With this proposal, it is now supported to declare a constructor in an
+extension, with the same syntax as in a class and in other type
+introducing membered declarations.
 
-[generic constructor proposal]: https://github.com/dart-lang/language/pull/4265
+*The semantic interpretation of constructors declared in extensions
+follows closely from the informal interpretation given above in which
+the constructor is viewed as a static method whose type parameters are
+the type parameters of the extension in which it is declared
+(including their bounds) and whose return type is the on-type of the
+extension (which may contain references to said type parameters).*
```

Comment:
(This worries me. It might be addressed later, but I'm worried *now* when reading this in order. The worry is that a generative constructor *is not* just a function. Invoking it means first introducing a new object, and it can be invoked as `const`.)